app/src/components/pages/Home.tsx

```diff
@@ -1,7 +1,40 @@
 import React from 'react';
+import uviclogo from 'images/uviclogo.png';
+import { Typography } from '@mui/material';

```

Comment:
Same as above - these can be combined or should be imported directly from the sub-directory to minimize bundle size.